### PR TITLE
Code Qaulity: Renamed ItemViewModel with ShellViewModel

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Summary

- Renamed ItemViewModel to ShellViewModel
- Renamed CurrentInstanceViewModel to ShellInstanceViewModel

## PR Checklist

- [ ] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Issue #14032
- [ ] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?
   _None_

## Screenshots

_None_

### CODE REVIEW

File: src/Files.App/Views/Shells/ColumnShellPage.xaml [hunk 1]
```diff
@@ -27,29 +27,7 @@
...
-			<RowDefinition Height="Auto" />
-		</Grid.RowDefinitions>
-		<Grid.ColumnDefinitions>
-			<ColumnDefinition Width="3*" />
-			<ColumnDefinition Width="Auto" />
-			<ColumnDefinition
-				x:Name="PreviewPaneColumn"
-				Width="2*"
-				MaxWidth="600" />
-		</Grid.ColumnDefinitions>
```
Comment (yaira2) [inline src/Files.App/Views/Shells/ColumnShellPage.xaml:52]: Where were these moved to?

Comment (0x5bfa) [reply src/Files.App/Views/Shells/ColumnShellPage.xaml:52]: Never used at all lol.
I'm not sure why this have been remained though.

Comment (yaira2) [reply src/Files.App/Views/Shells/ColumnShellPage.xaml:52]: Are you sure it's not referenced in code behind?

Comment (yaira2) [reply src/Files.App/Views/Shells/ColumnShellPage.xaml:52]: I took a look and can confirm this can be removed.

**Review by yaira2 — DISMISSED** [review]

**Pull request closed without merging**